readthedocs/urls.py

```diff
@@ -51,6 +51,9 @@
     url(r'^api/v2/', include('readthedocs.api.v2.urls')),
     # Keep `search_api` at root level, so the test does not fail for other API
     url(r'^api/v2/search/$', PageSearchAPIView.as_view(), name='search_api'),
+    # Deprecated
+    url(r'^api/v1/embed/', include('readthedocs.embed.urls')),
```

Comment:
don't think we want to keep the v1 endpoint, but the other repo has it.

Reply:
Yea, I think lets leave it for now. 👍